libraries/botframework-schema/src/teams/index.ts

```diff
@@ -1690,13 +1690,27 @@ export type Action = 'accept' | 'decline';
 
 /**
  * @interface
- * Specific details of a Teams meeting.
```

Comment:
For these interfaces, am I relying on DRY too much? There's basically two scenarios these cover:

1. `getMeetingInfo()`
2. meeting start/end events

They share similar base attributes (`MeetingDetailsBase`), but then differ. So, I've made them inherit from `MeetingDetailsBase`, mostly for the sake of DRY. Would it be better/simpler/backward-compat-proof to de-couple them more? One other benefit to de-coupling would be that if ported directly to .NET, fewer additional files are needed (.NET does a separate file for each class).

Reply:
It's probably fine for now, we can always decouple later if/when there are more differences between the types. The one thing would be taking care not to expose the Base type to users, which it looks like you handled below on line 1694.